css-fonts-3/support/css/font-style-matching.css

```diff
@@ -0,0 +1,1944 @@
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 100;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_onehundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 200;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_twohundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 300;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_threehundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_fourhundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 500;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_fivehundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 600;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_sixhundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 700;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_sevenhundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 800;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_eighthundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: normal;
+    font-weight: 900;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_normal_ninehundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: oblique;
+    font-weight: 100;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_oblique_onehundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
+    font-stretch: normal;
+    font-style: oblique;
+    font-weight: 200;
+    src: url("../fonts/matchingtest/CSSMatchingTest_normal_oblique_twohundred.ttf");
+}
+
+@font-face {
+    font-family: CSSMatchingTest;
```

Comment:
I don't quite understand this test. The important piece of the algorithm is what gets selected when the weight × width × slope space is sparse. It looks like you are populating the entire space with fonts, so I'm a little confused how these tests actually work.